lib/rubocop/cop/layout/indentation_width.rb

```diff
@@ -179,6 +176,24 @@ def check_members(base, members)
           end
         end
 
+        def select_check_member(member)
+          if member &&
+             member.children[0].is_a?(RuboCop::AST::Node) &&
```

Comment:
What are some cases where the first child would not be `nil` or `Node`? If there are such cases, we should add unit tests. If we can't think of any, this might be a bit defensive. 🙂

Reply:
I updated to the following fine approach.
https://github.com/rubocop-hq/rubocop/pull/6792#discussion_r260104653